src/platforms/ios/Podfile

```diff
@@ -1 +1 @@
-pod 'SSSnackbar', :git => 'https://github.com/EddyVerbruggen/SSSnackbar.git', :tag => '0.1.2.4'
\ No newline at end of file
+pod 'MaterialComponents/Snackbar'
```

Comment:
I think it's a good idea to add a version number so the plugin will not break when there is a new version with breaking changes.

Reply:
It's locked now.